netbox/dcim/tables/devices.py

```diff
@@ -670,11 +693,14 @@ class Meta(DeviceComponentTable.Meta):
             'cable', 'connection',
         )
         row_attrs = {
-            'class': get_interface_row_class,
             'data-name': lambda record: record.name,
             'data-enabled': get_interface_state_attribute,
+            'data-virtual': get_interface_virtual_attribute,
+            'data-mark-connected': get_interface_mark_connected_attribute,
+            'data-cable-status': get_interface_cable_status_attribute,
             'data-type': lambda record: record.type,
         }
+        cable_status_styles = [(slug, color) for slug, _, color in LinkStatusChoices.CHOICES]
```

Comment:
LinkStatusChoices is a tuple and not user-customizable.  We can likely just add the appropriate settings in the SCSS itself.

Reply:
Okay, wow, yeah, if LinkStatusChoices is not user-customizable then a lot of the complexity in this patch is entirely useless.

I'll see if I can pare it down.

Reply:
Yes, I can understand it is sometimes hard to tell what is and isn't user customizable.  I always take a look at the .CHOICES attribute on the choices class.  If it is a tuple (`(,)`) it is not customizable